test/e2e/kubernetes/kubernetes_test.go

```diff
@@ -568,7 +569,7 @@ var _ = Describe("Azure Container Cluster using the Kubernetes Orchestrator", fu
 				curlDeploymentName := fmt.Sprintf("ilb-test-deployment-%s", cfg.Name)
 				curlDeploy, err := deployment.CreateLinuxDeployIfNotExist("library/nginx:latest", curlDeploymentName, "default", "")
 				Expect(err).NotTo(HaveOccurred())
-				running, err := pod.WaitOnReady(curlDeploymentName, "default", 3, 30*time.Second, cfg.Timeout)
+				running, err := pod.WaitOnReady(curlDeploymentName, "default", 3, 1*time.Second, cfg.Timeout)
```

Comment:
why change the sleep time for these? should it also be a const?

Reply:
Should've mentioned this in the description. There's no reason to wait so long in between "check again for ready" tests. So, basically, while I was auditiong the "wait on ready" checks in the E2E code, I came across this and rationalized it.

Basically, decreasing the retry interval should speed up test runs. (The timeout until failure is untouched, so the only practical change is more kubectl commands against the apiserver).